about.html

```diff
@@ -0,0 +1,269 @@
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page" href="#"
+                ><i class="fab fa-facebook-f"></i
+              ></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="https://twitter.com/M_ayienga"
+                ><i class="fab fa-twitter"></i
+              ></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">English</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">My Page</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Logout</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="container-fluid second-nav">
+        <a class="navbar-brand hide-sm" href="home.html">
+          <span><i class="fas fa-archway"></i></span>
+          Historia Summit 2022</a
+        >
+        <button
+          class="navbar-toggler"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#navbarSupportedContent"
+          aria-controls="navbarSupportedContent"
+          aria-expanded="false"
+          aria-label="Toggle navigation"
+        >
+          <span class="navbar-toggler-icon">
+            <i class="menuIcon fas fa-bars"></i>
+            <i class="closeIcon fas fa-times"></i>
+          </span>
+        </button>
+        <div
+          class="collapse navbar-collapse justify-content-end"
+          id="navbarSupportedContent"
+        >
+          <ul class="navbar-nav mb-2 mb-lg-0 menu">
+            <li class="nav-item">
+              <a
+                class="nav-link active menuItem"
+                aria-current="page"
+                href="about.html"
+                >About</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active menuItem" aria-current="page" href="#"
+                >Program</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active menuItem" aria-current="page" href="#"
+                >Join</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active menuItem" aria-current="page" href="#"
+                >Sponsor</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active menuItem" aria-current="page" href="#"
+                >News</a
+              >
+            </li>
+            <li class="nav-item campaign">
+              <a class="nav-link active menuItem" aria-current="page" href="#"
+                >Campaigns</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <!-- WEBSITE CONTENT  -->
+
+    <div class="container-fluid main-content px-0">
+      <section class="introduction">
```

Comment:
- [ ] Kindly ensure that your introduction section is well centered in the top section of the page. It is currently more aligned to the right. Kindly fix it 👇 

![image](https://user-images.githubusercontent.com/63473584/181393425-b409a7bc-d5cb-499b-8792-4d9e057e9843.png)